web-client/src/views/MyAccount/VerifyNewEmailModal.tsx

```diff
@@ -9,7 +9,7 @@ export const VerifyNewEmailModal = connect(
     cancelSequence: sequences.clearModalSequence,
     closeVerifyEmailModalAndNavigateToMyAccountSequence:
       sequences.closeVerifyEmailModalAndNavigateToMyAccountSequence,
-    email: state.form.email,
+    email: state.user.pendingEmail,
```

Comment:
We don't always get here via a form now. I am verifying there are no issues here.